### ISSUE TITLE
[BUG] TweakScale is reporting

### ISSUE THREAD

**Issue by zer0Kerbal** [zer0Kerbal/ModularRocketSystems#147]
```
[LOG 22:35:12.710] [TweakScale] ERROR: **FATAL** Part NB0mLFOengine1 (MRS Liquid Fuel Engine, 0.625m "Sparkler") has a fatal problem due having duplicated properties - see issue [#34]( https://github.com/net-lisias-ksp/TweakScale/issues/34 ). at error:0
[LOG 22:35:12.714] [TweakScale] ERROR: **FATAL** Part NBfuelCell1m (MRS Inline Fuel Cell Array) has a fatal problem due having duplicated properties - see issue [#34]( https://github.com/net-lisias-ksp/TweakScale/issues/34 ). at error:0
```

**Comment by Codingale** [comment]
```
[LOG 23:41:17.581] [TweakScale] ERROR: **FATAL** Part mrs-tank-250-cone (MRS Fuel Tank, Adapter 2.5m to 1.25m) has a fatal problem due having duplicated properties - see issue [#34]( https://github.com/net-lisias-ksp/TweakScale/issues/34 ). at error:0
[LOG 23:41:17.581] [TweakScale] ERROR: **FATAL** Part NBconeTank2m (Kindly use new part numbers MRS Fuel Tank, Adapter 2.5m to 1.25m) has a fatal problem due having duplicated properties - see issue [#34]( https://github.com/net-lisias-ksp/TweakScale/issues/34 ). at error:0
```

Adding onto this, it seems it wasn't completely resolved, or maybe something strange happened since I'm using the release 1.13.99.0

**Comment by zer0Kerbal** [comment]
> Adding onto this, it seems it wasn't completely resolved, or maybe something strange happened since I'm using the release 1.13.99.0

Thank you @Codingale 

👀 investigating

@Lisias

**Comment by Lisias** [comment]
Classical case of double patching. @Codingale , please upload your **full** `KSP.log` and I will give it a look.

**Comment by Codingale** [comment]
Pardon the delay:
https://gist.github.com/Codingale/1d8e89f4dd7af9e393d277be8a70c37e

**Comment by Lisias** [comment]
@Codingale , I will need also the ModuleManager.log and the ConfigCache. I'm seeing you are using the /L Fork, nice to see I'm not the only crazy dude around! :)

```
[LOG 08:13:36.179] [ModuleManager] Version 4.2.2.5 /L
```

This makes things way easier. Just zip the `<KSP_ROOT>/PluginData/ModuleManager` folder with everything it have inside and drop the zip here on a comment.


On the bright side, MRS parching is being doing right:

https://github.com/zer0Kerbal/ModularRocketSystems/blob/f92610b8cb7d469c15fbf53acb0637f8b108672e/GameData/NecroBones/ModularRocketSystems/Compatibility/TweakScale.cfg#L163

It's a 3rd party. Humm.. Let me check TS deprecated patches….

**Comment by zer0Kerbal** [comment]
@Codingale - bug squashing continues. As soon as I know what the issue is, will publish a hotfix.

**Comment by Codingale** [comment]
Pardon the delay: [ModuleManager.zip](https://github.com/zer0Kerbal/ModularRocketSystems/files/11214369/ModuleManager.zip)
Anything else let me know.

**Comment by zer0Kerbal** [comment]
I believe I know the fix; will be back after some sleep.

**Comment by zer0Kerbal** [comment]
plan is to get this fixed and published in the next 36-48 hours.

**Comment by zer0Kerbal** [comment]
@Codingale @Lisias 
this should be the fix: [TweakScale-v1.3.99.1.zip](https://github.com/zer0Kerbal/ModularRocketSystems/files/11316851/TweakScale-v1.3.99.1.zip)


kindly try out and tell me the results
